scripts/download/tools.mjs

```diff
@@ -136,6 +136,23 @@ export default async function main(platform) {
   }
   await download(kimURL, kimPath, { expectedChecksum: kimSHA[0].split(/\s+/, 1)[0] });
 
+  if (kubePlatform === 'darwin') {
+    // On macos nerdctl is an existing resource, so we can connect our fabricated docker utility
+    // to it during tool installation. On Windows this happens during the build process.
+    const sourcePath = path.join(binDir, 'nerdctl');
+    const destPath = path.join(binDir, 'docker');
+
+    try {
+      await fs.promises.access(destPath);
+    } catch (err) {
+      if (err.code === 'ENOENT') {
+        await fs.promises.copyFile(sourcePath, destPath);
+      } else {
+        throw err;
+      }
+    }
```

Comment:
```suggestion
    await fs.promises.copyFile(sourcePath, destPath);
```
There's no reason we can't just always copy (especially if it's an old checkout and we changed the script, it'd be really confusing when the `docker` script isn't updated).

Reply:
Yeah, that was a holdover when `docker` was still a symlink. Those things are way more problematic than files; the `fs` library is kind of clunky when working with them.